2021/2021-09-07.md

```diff
@@ -0,0 +1,94 @@
+- Karina Kwiatek
+- Tyler Ward
+- Jake Howard
+- Tom Wheal
+- Will Barber
+- Andy Barrett-Sprot
+
+
+## Agenda
+
+1. Structure of Kit Team and priorities for the next year
+2. Recovery of missing kit from SR2017, 2019 and 2020.
+3. Asset Audit
+4. Feasibility of shipping physical kit for SR2022
+5. Kit Software progress and plan (ex-Brain Team work)
+6. Documentation (i.e deprecation of Kitbook)
+7. Regular meeting slot going forward
+8. Blog Posts for website
+
+## Topics
+
+- Recovery of missing kit from SR2017, 2019 and 2020.
+    - 4 / 6 kits from schools that returned are in Tom's Garage
+    - Been unable to reach 14 schools - staff retiring
+    - 18 kits are being kept by school. Safe to assume that they're good or can send spares
+    - 8 kits in Tom's garage
+    - Morse suggested sending signed letters to headmaster
+    - Ask teams holding on to kits to audit them?
+    - No hard number today
+
+- Feasibility of shipping
+    - We could drop vision. It would not be a huge amount of work
+    - We've been able to give an alternative instead. Without vision how would they navigate?
+    - No vision = more exciting lighting
+    - Alternative = lots of ultrasound sensors
+    - Off the shelf robots = money down the drain
+        - Unless can resell or recoup money
+        - Use as prizes?
+    - No courier will accept existing batteries as they have not certification.
+    - Should we be taking the risk of sending out uncertified batteries?
+    - LiPos are possibly not storage charged, although not many teams have touched their kit?
+    - CTC are open to suggestions re: what we ship
+        - Maybe multiple regional competitions?
+
+- Asset Audit
+    - There's a lot of kit with old volunteers including RUBs
+    - Most stuff is marked with an unknown state
+    - All of the old kit is in there (v3)
+        - We probably can't sell this now
+        - Internals of boards are heavily bespoke
+        - We probably won't ship these as kits again
+        - Could maybe be broken down for parts
+
+- Inventory software
+    - Needs improving or replacing
+    - Not necessarily our decision
+
+- Runbook
+    - Documentation should be in the same place. No fragmentation
+    - Wiki used to work well
+        - Easy to edit and update
+        - Frictionless editing - no review process
+        - More likely to be up to date
+        - Using Mediawiki, you can link to previous revisions
+            - Can be painful to look through
+    - Historical docs should be archived
+        - Useful in git history
+        - Mostly useless - structure has changed
+        - Reference for previous years is useful
+        - maybe link at bottom of current page
+    - Git is hard to use, especially for non-technical people
+
+- Blog posts
+    - Embedded Software in Rust
+
+- Unplanned: Recruitment
+    - Regional Competitions are a great place to recruit people
+
+- Meeting Time
+    - We agreed to meet at 7pm every other Tuesday for now.
```

Comment:
As a general concern -- deciding a meeting time while at a meeting happening at that time is prone to bias since people who can't make that slot won't be present to tell you that. Suggest as a general principle that we (all volunteer teams) ensure that the decision mechanism accounts for the views of those not present.